doc/tutorials/external-commands.md

```diff
@@ -0,0 +1,51 @@
+# How-to: Add external commands
+
```

Comment:
Please start with some introduction what the tutorial is going to tell us.

Reply:
done